ts/store/middlewares/analytics.ts

```diff
@@ -417,6 +420,7 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
       }
       break;
     case getType(removeAccountMotivation):
+    case getType(deleteUserDataProcessing.request):
```

Comment:
Can you track also the `deleteUserDataProcessing.success` and the `deleteUserDataProcessing.failure` actions?